src/main/scala/com/normation/plugins/datasources/Scheduler.scala

```diff
@@ -124,8 +126,13 @@ class DataSourceScheduler(
     cancel()
     // actually start the scheduler by subscribing to it
     if(datasource.enabled) {
-      DataSourceLogger.debug(s"Scheduling runs for data source with id '${datasource.id.value}'")
-      scheduledTask = Some(source.subscribe())
+      if(pluginStatus.isEnabled) {
+        DataSourceLogger.debug(s"Scheduling runs for data source with id '${datasource.id.value}'")
+        scheduledTask = Some(source.subscribe())
+      } else {
+        // the plugin is disabled, does nothing
+        DataSourceLogger.warn(s"The datasource with id '${datasource.id.value}' is enabled but the plugin is disabled (reason: ${pluginStatus.enabledStatus}. Not scheduling future runs for it.")
```

Comment:
so we'll need to restart the webapp in this case if we correct the enabling file ?

Reply:
Well, not because of that because you could just re-enable the plugin in the UI or query update from API, and it would restart the scheduler. 
But the license file is only read at boot time. That seems an  acceptable tradeoff for performance and complexity for a minimal v1.

Actually, I could time time spent in the license evaluation to see if it matches the timeframe of datasources and do it each time. That would also mean that each plugin may ends with a different strategy. 

Actually, the correct situation is to have Rudder provides a proxy for licenses with cache, and an update command which read again license. On v2 :)